manifest-build-tools/application/version_generator.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+# Copyright 2015-2016, EMC, Inc.
+
+"""
+The script compute the version of a package, just like:
+1.1-1-devel-20160809150908-7396d91
+
+usage:
+./on-tools/manifest-build-tools/HWIMO-BUILD on-tools/manifest-build-tools/application/version_generator.py 
+--repo-dir /home/onrack/rackhd/release/rackhd-repos/PengTian0/b/b/on-http
+--manifest-repo-dir /home/onrack/rackhd/release/rackhd-repos/PengTian0/b/build-manifests
+--is-official-release
```

Comment:
The parameter type is Boolean.
If command with ```--is-official-release```, that means the parameter is true.
If command without it, that means the parameter is false.